count_cells.py

```diff
@@ -7,28 +7,32 @@
 import sys
 from typing import (
     Sequence,
+    Mapping,
     Union,
 )
 
 from util import (
+    generate_project_uuid,
     get_target_project_dirs,
     open_maybe_gz,
 )
 
 
 class CountCells:
-    cell_counts_file = Path('cell_counts.json')
 
     def __init__(self, argv):
         logging.basicConfig(level=logging.INFO)
         parser = argparse.ArgumentParser(description=__doc__)
         group = parser.add_mutually_exclusive_group(required=True)
         group.add_argument('--list', '-l',
-                           help='list cell count values',
+                           help='List cell count values',
                            action='store_true')
         group.add_argument('--write', '-w',
-                           help='write cell count files',
+                           help='Write cell count files',
                            action='store_true')
+        parser.add_argument('--accession', '-a', type=str,
```

Comment:
Please remove. Exploit the `SKUNK_ACCESSIONS` code from `util.py`. Heads up: most of the argparse boiler plate is going to go away soon.